docs/nomad-audit-2025-05-20.md

```
@@ -0,0 +1,43 @@
+# Nomad Audit Report - 2025-05-20
+
+## Executive Summary
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)